build/BranchInfo.props

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <MajorVersion>0</MajorVersion>
     <MinorVersion>10</MinorVersion>
-    <PatchVersion>0</PatchVersion>
+    <PatchVersion>1</PatchVersion>
```

Comment:
Let's be sure to change this back when we merge master into release for `0.11.0`.